homeassistant/components/envisalink/__init__.py

```diff
@@ -175,6 +179,12 @@ def partition_updated_callback(data):
         _LOGGER.debug("The envisalink sent a partition update event")
         async_dispatcher_send(hass, SIGNAL_PARTITION_UPDATE, data)
 
+    @callback
+    def zone_bypass_update(data):
```

Comment:
```suggestion
    def async_zone_bypass_update(data):
```

Reply:
Usually we prefix functions that run in the event loop with `async_` 

Since everything else is already done without it, its fine to do in a followup PR

Reply:
There are already other changes in this PR which clean up some similar inconsistencies so I made these changes here as well.